### DISCUSSION TITLE
How to add starting items?

### DISCUSSION THREAD

**Discussion by ronderksen** [Lurkars/gloomhavensecretariat#277] [Q&A] [answered]
We started our Frosthaven campaign yesterday. We used the X-Haven app for the first scenario, but this app seems to offer more functionality so I'm trying to figure out how to use it properly. Most things are very clear and work great. But I can't figure out how to add the starting items to our characters. 

If I look at the character sheet before starting a scenario, I can see they get 30 gold (which is not correct btw). But there's no way to add items. Also after finishing the first scenario (01) there is no option to add the items, only items that are available for crafting.

Is this an oversight in the app, or am I missing something?

**Comment by Lurkars** [comment]
I will take a look. The starting items are definitely missing as initial available will fix that. With the 30 gold, the only change I read to this is, that every gold not spent on items on start is lost. So will check, how I can add this. Anyways, as quack workaround is you really need the items now: in party/campaign sheet click on the "< Campaign Sheet" to go to the backside, there you can add "Unlocked Items" and just unlock manually what you need!

**Comment by Lurkars** [comment, marked as answer]
I added now the starting items in [v0.62.7](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.62.7). Also when FH is set as edition, the mentioned GH items are also available. What I won't fix (for now) is that remaining of the 30 gold. This has to be done manually by now.
